### DISCUSSION TITLE
ESP32-2432s022 ESPHome

### DISCUSSION THREAD

**Discussion by dirtyharryiv** [witnessmenow/ESP32-Cheap-Yellow-Display#198] [General]
Does somebody have this version and got it running with esphome? I only have a white screen with the provided examples.
The display is a little bit smaller but other than that it looks like a CYD

**Comment by mstewart197930** [comment]
I have a similar issue on a different project, it would seem these are a different display chip and pin out, to the slightly bigger ones.

**Comment by guybw** [comment]
Just to add, I have the same issue.

**Comment by nielsnl68** [comment]
Please ping me on the esphome discord and i will try to see what i can do for you.

**Comment by nielsnl68** [comment]
Okay, it looks that they are using the Parallel interface (i80) between display and esp32.
At this moment in time there is no support in esphome yet. We are working to get this implemented very soon.

**Comment by dirtyharryiv** [comment]
https://github.com/rzeldent/esp32-smartdisplay
This Library supports the display. Maybe it will help you :-)

**Comment by chitkosarvesh** [comment]
Hello, is there any update on this?
I am looking to configure this board with ESPHome on my Homeassistant and I am just stuck on a white screen.
LVGL is also not able to run.
